_posts/insights_0_transcription.adoc

```diff
@@ -0,0 +1,233 @@
+http://www.youtube.com/watch?v=YKm8rVzLhNE&t=00m03s[00:03]:
+
+**Emmanuel**: Ready, we're doing this thing, "Okay yes, Quarkus inside number zero inception.
+
```

Comment:
not reviewing further, sort on time.

Reply:
Yeah yeah - I'm not planning on fine checking the transcripts. If grave things Goes in prs can be done later.